src/bin/pg_verifybackup/parse_manifest.c

```diff
@@ -119,7 +119,7 @@ void
 json_parse_manifest(JsonManifestParseContext *context, char *buffer,
 					size_t size)
 {
```

Comment:
are changes to this file here intentional or accidental?

Reply:
yes, json changes are required.